pkg/engine/requests_test.go

```diff
@@ -471,3 +471,175 @@ func TestGameEngineSendGetMidGameScoreBatchAtMidGameReturnsExpectedScores(t *tes
 		t.Fatalf("Player 2 score: %#v, expected: %#v", midGameScoreResp.Scores[2], expected)
 	}
 }
+
+func TestPlaySingleTurnGameWithNoLogger(t *testing.T) {
+	engine, err := StartGameEngine(4, "")
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	tileset := tilesets.TileSet{
+		Tiles:        []tiles.Tile{tiletemplates.StraightRoads()},
+		StartingTile: tiletemplates.SingleCityEdgeStraightRoads(),
+	}
+
+	game, err := engine.GenerateGame(tileset)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	ptile := elements.ToPlacedTile(game.Game.CurrentTile)
+	ptile.Position = position.New(1, 0)
+	ptile.GetPlacedFeatureAtSide(side.Right, feature.Road).Meeple = elements.Meeple{PlayerID: elements.ID(1), Type: elements.NormalMeeple}
+
+	playTurnRequest := &PlayTurnRequest{
+		GameID: game.ID,
+		Move:   ptile,
+	}
+	playTurnResp := engine.SendPlayTurnBatch([]*PlayTurnRequest{playTurnRequest})[0]
+	if playTurnResp.err != nil {
+		t.Fatal(playTurnResp.err.Error())
+	}
+
+	expectedScore := uint32(2)
+	if playTurnResp.FinalScores[elements.ID(1)] != expectedScore {
+		t.Fatalf("Expected score of player1: %#v, got: %#v", expectedScore, playTurnResp.FinalScores[elements.ID(1)])
+	}
+}
+
+func TestGetLegalMovesAndPlayItWithPlayTurnRequestWithNoLogger(t *testing.T) {
+	engine, err := StartGameEngine(4, "")
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	game, err := engine.GenerateGame(tilesets.StandardTileSet()) // Preferably change for GenerateSeededGame when benchmarks are merged
```

Comment:
I'd suggest adding a `TODO` in this comment, so that it's easier to find and we don't forget it in the future.
Same thing with the comment in line 544.

Reply:
TODOed